starter-apps/vue/vue-template/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "gcds-vue-start-app",
```

Comment:
Sorry nit-picky but I wanted to double-check: is it correct that we are using `start-app` here or should it be `starter-app`?

Reply:
No prob, I was wondering about that too but you mentioned `gcds-vue-start-app` [here](https://github.com/cds-snc/gcds-examples/pull/21#discussion_r1756003231) and I simply copied it lol!

Reply:
Haha oops, sorry about that typo!